sparkle/src/components/DataTable.tsx

```diff
@@ -243,6 +248,24 @@ DataTable.Row = function Row({
           <Icon visual={MoreIcon} size="sm" />
         </td>
       )}
+      {moreMenuItems && (
```

Comment:
Let's get rid of `onMoreClick` then so there is only one way to add this more menu.